index.html

```diff
@@ -35,7 +35,9 @@ <h2>Create your own motivational poster</h2>
   </section>
   <section class="saved-posters hidden">
     <h2>Saved Posters</h2>
-    <article class="saved-posters-grid"></article>
+    <article class="saved-posters-grid">
```

Comment:
Nothing changed here except for adding whitespace. I would change this line back to the original so it doesn't look like a diff.